packages/image_picker/image_picker_for_windows/example/lib/main.dart

```diff
@@ -0,0 +1,465 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
```

Comment:
We don't use the app-facing platform in implementation examples (there are some legacy violations, but we don't do it any more), as it creates a circular dependency that causes problems for testing changes. This should instead be written directly against `image_picker_platform_interface`.